### PR TITLE
Smallpatches

### PR DESCRIPTION
**Pull request by InfusOnWoW**

### CODE REVIEW

File: WeakAuras/SubRegionTypes/BarModel.lua [hunk 1]
```diff
@@ -65,58 +134,55 @@ end
...
+  region.parent = parent
 
-  if (data.api) then
-    region.model:SetTransform(data.model_st_tx / 1000, data.model_st_ty / 1000, data.model_st_tz / 1000,
-      rad(data.model_st_rx), rad(data.model_st_ry), rad(data.model_st_rz), data.model_st_us / 1000);
-  else
-    region.model:ClearTransform();
-    region.model:SetPosition(data.model_z, data.model_x, data.model_y);
-  end
+  -- TODO remove
```
Comment (Stanzilla) [inline WeakAuras/SubRegionTypes/BarModel.lua]: now or later?

Comment (InfusOnWoW) [reply WeakAuras/SubRegionTypes/BarModel.lua]: Now, I needed that to debug.

**Comment by mrbuds** [comment]
Getting this error when i add a model subregion to an icon
```
1x WeakAuras\SubRegionTypes\BarModel.lua:146: Usage: <unnamed>:SetModelAlpha(alpha)
[string "=[C]"]: in function `SetModelAlpha'
[string "@WeakAuras\SubRegionTypes\BarModel.lua"]:146: in function `SetVisible'
[string "@WeakAuras\SubRegionTypes\BarModel.lua"]:194: in function `modify'
[string "@WeakAuras\RegionTypes\RegionPrototype.lua"]:543: in function `modifyFinish'
[string "@WeakAuras\RegionTypes\Icon.lua"]:541: in function `modify'
[string "@WeakAuras\WeakAuras-@project-version@.lua"]:4463: in function `SetRegion'
[string "@WeakAuras\WeakAuras-@project-version@.lua"]:4364: in function <WeakAuras\WeakAuras.lua:4267>
[string "@WeakAuras\WeakAuras-@project-version@.lua"]:4394: in function `Add'
[string "@WeakAurasOptions\WeakAurasOptions-@project-version@.lua"]:2592: in function `func'
[string "@WeakAuras\WeakAuras-@project-version@.lua"]:129: in function `ApplyToDataOrChildData'
[string "@WeakAurasOptions\WeakAurasOptions-@project-version@.lua"]:2599: in function <WeakAurasOptions\WeakAurasOptions.lua:2598>
[string "@WeakAurasOptions\WeakAurasOptions-@project-version@.lua"]:2630: in function <WeakAurasOptions\WeakAurasOptions.lua:2629>
[string "=[C]"]: ?
[string "@Ace3\AceConfig-3.0-3\AceConfigDialog-3.0\AceConfigDialog-3.0-78.lua"]:51: in function <...nfig-3.0\AceConfigDialog-3.0\AceConfigDialog-3.0.lua:49>
[string "@Ace3\AceConfig-3.0-3\AceConfigDialog-3.0\AceConfigDialog-3.0-78.lua"]:843: in function <...nfig-3.0\AceConfigDialog-3.0\AceConfigDialog-3.0.lua:664>
[string "=[C]"]: ?
[string "@Quartz\libs\AceGUI-3.0\AceGUI-3.0-40.lua"]:72: in function <Quartz\libs\AceGUI-3.0\AceGUI-3.0.lua:70>
[string "@Quartz\libs\AceGUI-3.0\AceGUI-3.0-40.lua"]:290: in function `Fire'
[string "@Ace3\AceGUI-3.0-40\widgets\AceGUIWidget-Button.lua"]:22: in function <...dOns\Ace3\AceGUI-3.0\widgets\AceGUIWidget-Button.lua:19>
```

**Comment by InfusOnWoW** [comment]
Ah right, fixed.

**Pull request merged**